src/components/NavBar.scss

```diff
@@ -0,0 +1,74 @@
+@import 'src/styles/settings/general';
+
+.navBar {
+    position: fixed;
+    bottom: 0;
+    width: 100%;
+    height: 60px;
+    background-color: white;
+    box-shadow: 0 -2px 3px rgba(99, 99, 99, 0.75);
+}
+
+.navLeft {
+    position: absolute;
+    left: 0;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    border: none;
+    height: 100%;
+    width: 50%;
+    padding-right: 50px;
+}
+
+.navRight {
+    position: absolute;
+    right: 0;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    border: none;
+    height: 100%;
+    width: 50%;
+    padding-left: 50px;
+}
+
+.navButton {
+    height: 100%;
+    width: 40px;
+    margin-left: 10px;
+    margin-right: 10px;
+    border: none;
+    outline: none;
+    padding: 0;
+    background: transparent;
+
+    img {
+        width: 20px;
+        height: 20px;
+        margin: 0 10px;
+    }
+
+    span {
+        background-size: contain;
+        color: $text-color-light;
+        font-size: 8px;
+    }
+}
+
+.kudoButton {
+    position: absolute;
+    left: 50%;
+    height: 105px;
+    width: 105px;
+    margin-left: -52.5px;
+    margin-top: -24px;
+    z-index: 100;
```

Comment:
To avoid z-index hell, you could create a list of all stuff that needs to be stacked, so all z-indexes are in one file. (See for example https://github.com/kabisa/hue-labs-frontend/blob/master/app/styles/settings/_general.scss)